### PR TITLE
Add config file for management script

### PR DESCRIPTION
**Pull request by mbernasocchi**

### CODE REVIEW

**Comment by mbernasocchi** [comment]
@tgxn might be useful to allow cleaner separation of settings an the script

**Comment by tgxn** [comment]
Hey mate, I'm a huge fan of this approach, much better to use a config file!

There's reference to [an `xmm7360.ini` file here](https://github.com/xmm7360/xmm7360-pci/blob/master/rpc/open_xdatachannel.py#L15), maybe we can just remove the need to use the `--apn` cli arg here altogether?

Looking at [configargparse](https://pypi.org/project/ConfigArgParse/), can we add a `.sample` file with some sensible defaults that people can modify/un-comment as appropriate?
```
# used by the setup script
BIN_DIR=/usr/local/bin

# driver config
apn=your.apn.here
ip-fetch-timeout=1

# noresolv
# nodefaultroute
```

**Comment by tgxn** [comment]
This would help with having to modify command-line params for dbus support too: https://github.com/xmm7360/xmm7360-pci/pull/66#issuecomment-788673520 🎉

File: scripts/lte.sh [hunk 1]
```diff
@@ -59,6 +65,6 @@ fi
 if [[ "$1" = "up" ]]; then
   echo "bringing wwan0 up!" 
 
-  python3 $SCRIPT_DIR/../rpc/open_xdatachannel.py --apn $CONNECT_APN
+  python3 $SCRIPT_DIR/../rpc/open_xdatachannel.py --apn $apn
```
Comment (tgxn) [inline scripts/lte.sh:68]: should be ok without `--apn` if the config file is being read correctly.

**Comment by mbernasocchi** [comment]
@tgxn done refactoring, you can review now :)
the parameters are all set in xmm7360.ini and so it should also help for #66, I also updated a bit the readme

**Comment by tgxn** [comment]
Looks good! Cheers!

**Pull request merged**